functions/find.bash

```diff
@@ -196,7 +196,7 @@ setup_monitor_mode() {
       echo "$(timestamp) [openHABian] Monitor Mode patch has not been updated for kernel version 5.4 on your RPi... CANCELED"
       return 0
     elif is_pithreeplus || is_pifour; then
-      firmwareVersion="bcm43455c0/7_45_189"
+      firmwareVersion="bcm43455c0/7_45_206"
```

Comment:
uhh. Now I don't know the context but static firmware strings are not an optimum solution

Reply:
It is required because of the way that nexmon does their versioning and patches. Dynamic firmware would result in issues because they only support certain versions.

Reply:
just sayin' . I'm sure you would have chosen a better means if it existed.

Reply:
Believe me, I wanted to. I just couldn't find any way to do it with the way nexmon does their patches.